edc-extensions/dataplane-proxy/edc-dataplane-proxy-provider-api/src/main/java/org/eclipse/tractusx/edc/dataplane/proxy/provider/api/DataPlaneProxyProviderApiExtension.java

```diff
@@ -74,6 +94,19 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
+        var port = context.getSetting(PROVIDER_PORT, DEFAULT_PROVIDER_PORT);
```

Comment:
why are these changes necessary? Your PR is dealing with packaging for - and publishing to - docasaurus. Why are code-level changes needed here? 
And if they really are necessary they should be clearly explained in the PR description. 

In any event, the `CONSUMER_CONFIG_KEY` and `CONSUMER_API_ALIAS` constants are undefined.

Reply:
Yeah, this does seem lie a rebase issue. I'll look into that tomorrow.